### PR TITLE
Create Send File to Webhook

### PR DESCRIPTION
**Pull request by shootdot**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by TheRealPikachu** [comment]
👀

File: actions/get_image_from_hex_code_MOD.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    Store Image In:<br>
+    <select id="storage" class="round">
+      ${data.variables[1]}
+    </select>
+  </div>
+  <div id="varNameContainer" style="float: right; width: 60%;">
+    Variable Name:<br>
+    <input id="varName" class="round" type="text">
+  </div>
+</div>`
```
Comment (almeidx) [inline actions/get_image_from_hex_code_MOD.js:39]:
```suggestion
<div>
  <div style="width: 90%;">
    Image Hex Code (Remove # from Hex code):<br>
    <input placeholder="ffffff" id="input" class="round" type="text">
  </div><br>
  <div>
    Size of Image:<br>
    <input placeholder="1024x1024" id="size" class="round" type="text">
  </div>
</div><br>
<div style="padding-top: 8px;">
  <div style="float: left; width: 35%;">
    Store Image In:<br>
    <select id="storage" class="round">
      ${data.variables[1]}
    </select>
  </div>
  <div id="varNameContainer" style="float: right; width: 60%;">
    Variable Name:<br>
    <input id="varName" class="round" type="text">
  </div>
</div>`
```

File: actions/send_file_to_webhook_MOD.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  subtitle (data) {
+    return 'Send a file to a webhook'
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
+    const dataType = 'Number'
+    return ([data.varName, dataType])
```
Comment (almeidx) [inline actions/send_file_to_webhook_MOD.js:13]:
```suggestion
    if (parseInt(data.storage) !== varType) return
    return ([data.varName, 'Number'])
```

File: actions/send_file_to_webhook_MOD.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    const options = {
+      files: [
+        this.getLocalFile(this.evalMessage(data.file, cache))
+      ]
+    }
+    if (!webhook) {
+      this.callNextAction(cache)
+      return
+    }
+    webhook.send(options)
```
Comment (almeidx) [inline actions/send_file_to_webhook_MOD.js:59]:
```suggestion
    if (!webhook) {
      this.callNextAction(cache)
      return
    }
    webhook.send({
      files: [
        this.getLocalFile(this.evalMessage(data.file, cache))
      ]
    })
```

**Comment by shootdot** [comment]
> 
> 
> I don't think Get Image from Hex Code is necessary.

One of the member asked, so I made....

**Comment by RigidStudios** [comment]
Isn't the hex code image possible with Canvas mods already?

File: actions/send_file_to_webhook_MOD.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    if (!webhook) {
+      this.callNextAction(cache)
+      return
+    }
+    webhook.send({
+      files: [
+        this.getLocalFile(this.evalMessage(data.file, cache))
+      ]
+    })
+    this.callNextAction(cache)
```
Comment (almeidx) [inline actions/send_file_to_webhook_MOD.js:57]:
```suggestion
    }).then(() => this.callNextAction(cache))
```

**Comment by shootdot** [comment]
> * Remove the Get Image from Hex Code action
> * Add this to the docs

Ok

**Pull request merged**